sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/Samples/Sample02_Serialization.cs.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Serialization;
+using Azure.Core.TestFramework;
+using Azure.Data.SchemaRegistry.Serialization;
+using Azure.Data.SchemaRegistry.Tests.Serialization;
+using Azure.Messaging;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Producer;
+using NUnit.Framework;
+using TestSchema;
+
+namespace Azure.Data.SchemaRegistry.Tests.Samples
+{
+    public class Sample02_Serialization : SamplesBase<SchemaRegistrySerializerTestEnvironment>
```

Comment:
It would be nice to include a sample where we use a real JsonSchema library.

Reply:
I've asked that we do not do that and stick to a fake implementation.  There's no widely supported and available option - either we're implicitly recommending that customers use the commercial Newtonsoft offering or we risk being perceived as endorsing a small one-person run open-source project.